pkg/kev/converter/kubernetes/utils.go

```diff
@@ -975,11 +961,6 @@ func ToUnstructured(o runtime.Object) (map[string]interface{}, error) {
 	return raw, nil
 }
 
-// FromUnstructured converts unstructured to target object
-func FromUnstructured(unstructured map[string]interface{}, target interface{}) error {
```

Comment:
Wasn't this func used? It could have been used downstream perhaps?

Reply:
Nope. No where.

Unless `utils.go` has an external entity calling it - nothing was using `FromUnstructured `.

Reply:
After more research nothing uses it.